pkg/project/auth/cache.go

```diff
@@ -242,6 +281,36 @@ func (ac *AuthorizationCache) synchronizePolicyBindings(userSubjectRecordStore c
 	}
 }
 
+// synchronizePolicies synchronizes access over each policy
+func (ac *AuthorizationCache) synchronizeClusterPolicies(userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) {
+	items := ac.clusterPolicyIndexer.List()
+	for i := range items {
+		policy := items[i].(*authorizationapi.ClusterPolicy)
+		reviewRequest := &reviewRequest{
+			namespace:                  policy.Namespace,
+			policyUIDToResourceVersion: map[types.UID]string{policy.UID: policy.ResourceVersion},
+		}
+		if err := ac.syncHandler(reviewRequest, userSubjectRecordStore, groupSubjectRecordStore, reviewRecordStore); err != nil {
```

Comment:
I now see no point in doing this operation, or anything in this function.  The whole point to synchronize against these is to get an updated reviewRecord for a specific namespace.  Cluster policies are not in a namespace, so calling syncRequest for a no named namespace is meaningless.  Get rid of this.